ajax/libs/hammer-time/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "hammer-time",
+  "filename": "dist/hammer-time.min.js",
+  "description": "A fast click based on the touch-action css property",
+  "version": "1.1.0",
+  "author": "Alex Schmitz",
+  "homepage": "https://hammerjs.github.io/",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/hammerjs/hammer-time.git"
+  },
+  "license": "MIT",
+  "keywords": [
+    "touch-action",
+    "touch",
+    "action",
+    "css",
+    "touchevents",
+    "events",
+    "pointerevents",
+    "pointer",
+    "click",
+    "delay",
+    "hammer"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/hammerjs/hammer-time.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "hammer-time.js",
+          "dist/hammer-time.min.js"
+        ]
+      }
+    ]
+  }
```

Comment:
This is not the good way, we just need `hammer-time.min.js` and `hammer-time.js` files and we don't need the folder `dist`.
Please use as the followings
```js
  "autoupdate": {
    "source": "git",
    "target": "git://github.com/hammerjs/hammer-time.git",
    "fileMap": [
      {
        "basePath": "",
        "files": [
          "hammer-time*"
        ]
      },
      {
        "basePath": "dist",
        "files": [
          "hammer-time*"
        ]
      }
    ]
  }
```

It let the file structure better
from 
```
├── 1.1.0
│   ├── dist
│   │   └── hammer-time.min.js
│   └── hammer-time.js
```
to
```
├── 1.1.0
│   ├── hammer-time.js
│   └── hammer-time.min.js
```
Note: 
We usually use `**/*` under `dist` folder, but in this case, if use `**/*` it will get the `arraysweeper.min.js` file in old versions.
I think we don't need to add this file, so I use `hammer-time*`